### PR TITLE
Feature/jcs/heading

### LINKED ISSUE #10: Heading component
### Feature Categories:

- [x] Angular
- [x] SCSS / Styling
- [x] Testing
- [x] UI 
- [ ] BugFix

**Describe the feature/Bug your gonna work on**
- Setup header component inside app component.
- Follow wire frame
- Setup html and scss
- scss setup global variables 
- scss add mixins

**Additional context**
Add any other context or screenshots about the feature.

### LINKED ISSUE #14: SCSS File Architecture
### Feature Categories:

- [ ] Angular
- [x] SCSS / Styling
- [ ] Testing
- [ ] UI 
- [ ] BugFix

**Describe the feature/Bug your gonna work on**

- Add global variables, mixins, animations
- Import all partials to root/style.scss for easy importing into component style sheets
- Go over with team walkthrough file structure
- Add additional color schemes 
- Clean SCSS file structure

**Additional context**
Add any other context or screenshots about the feature.

### PR DESCRIPTION
**Pull request by JoshSevy**
# Turing Index Pull Request Form

## Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

-Add header component
- Add styling to main/landing page
- SCSS file architecture
- Hamburger menu with nav links 
- Animated page change for nav links
- followed a blueprint/schematic theme

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Testing (add-testing to project)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain: File Architecture 

Setup clean file tree for style elements
This was more experimenting with Angular and SCSS

## Has This Been Tested?
- [ ] No
- [x] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
Per installed angular tests component passed render tests, tested google chrome browser
# Message for reviewer:

You are so great!

# Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -123,6 +123,10 @@
...
       }
-    }},
-  "defaultProject": "web-app"
-}
+    }
+  },
+  "defaultProject": "web-app",
+  "cli": {
+    "analytics": false
+  }
```
Comment (NicoleLatifi) [inline angular.json:131]: What does this do? Does it just mean we're not sharing Angular CLI usage data with Google Analytics? https://angular.io/cli/usage-analytics-gathering

Comment (JoshSevy) [reply angular.json:131]: Its the Angular cli feedback I chose no feedback to angular when creating a component.

Comment (JoshSevy) [reply angular.json:131]: [Angular cli analytics docs](https://angular.io/cli/analytics)

File: src/app/map/map.component.scss [hunk 2]
```diff
@@ -1,4 +1,11 @@
-// svg {
-//   display: block;
-//   width: 100%;
-// }
\ No newline at end of file
+.st0{fill:none;stroke:#eee;stroke-width:4;stroke-miterlimit:5;}
```
Comment (NicoleLatifi) [inline src/app/map/map.component.scss:1]: Thank you for moving the styling over to this scss file.

**Review by NicoleLatifi — COMMENTED** [review]
This all looks amazing. I'm excited to see what all of the styling animations do and get an understanding of the setup involved with Angular testing.

File: src/app/app.component.html [hunk 3]
```diff
@@ -1 +1,6 @@
-<app-map></app-map>
\ No newline at end of file
+<app-heading></app-heading>
```
Comment (Leasw144) [inline src/app/app.component.html:1]: Do Angular components not need to be wrapped in one div like it does in React? For example I would've thought it needed to look liked this

```
<div>
  <app-heading></app-heading>
  <div class="map-container">
    <app-map></app-map>
  </div>
</div>
```
Or am I missing something?

**Review by Leasw144 — APPROVED** [review]
You are amazing.

File: src/app/app.component.html (hunk 3 above)
Comment (JoshSevy) [reply src/app/app.component.html:1]: I do not believe it does since it is compiled into a body under the hood.

**Pull request merged**